pep8.py

```diff
@@ -2061,7 +2061,8 @@ def read_config(options, args, arglist, parser):
         # Second, parse the configuration
         for opt in config.options(pep8_section):
             if opt.replace('_', '-') not in parser.config_options:
-                print("  unknown option '%s' ignored" % opt)
+                if options.verbose:
+                    print("  unknown option '%s' ignored" % opt)
```

Comment:
Alack no, I tested it out locally last week and it didn't seem to override the config option. 

Requires a bit more digging, but seems on right track. 

We use `--quiet` in a couple of other engines I think, including brakeman.